modules/bbp-morphology/src/main/resources/schemas/bbp/morphology/annotatedslice/v0.1.0.json

```diff
@@ -0,0 +1,66 @@
+{
+  "@context": [
+    "{{base}}/contexts/nexus/shacl/rec-shacl-20170720/v0.1.0",
+    "{{base}}/contexts/nexus/prov/rec-prov-o-20130430/v0.1.0",
+    "{{base}}/contexts/bbp/core/entity/v0.1.0",
+    "{{base}}/contexts/nexus/core/standards/v0.1.0",
+    "{{base}}/contexts/bbp/neurosciencegraph/core/v0.1.0",
+    {
+      "bbpentsh": "{{base}}/schemas/bbp/core/entity/v0.1.0/shapes/",
+      "bbpqsh": "{{base}}/schemas/bbp/core/quantitativevalue/v0.1.0/shapes/",
+      "bbppsh": "{{base}}/schemas/bbp/core/person/v0.1.0/shapes/",
+      "this": "{{base}}/schemas/bbp/experiment/annotatedslice/v0.1.0/shapes/"
+    }
+  ],
+  "@type": "owl:Ontology",
+  "imports": [
+    "{{base}}/schemas/bbp/core/entity/v0.1.0",
+    "{{base}}/schemas/bbp/core/quantitativevalue/v0.1.0"
+  ],
+  "shapes": 
+    [
+      {
+        "@id": "this:AnnotatedSliceShape",
+        "@type": "sh:NodeShape",
+        "description": "Annotated slice shape definition. An annotated slice should have an IRI as identifier",
```

Comment:
I would suggest to remove this as we set "nodeKind": "sh:BlankNodeOrIRI".

Reply:
OK updated